clients/privacy-center/public/fides-js-components-demo.html

```diff
@@ -282,9 +282,11 @@ <h1>fides.js demo page</h1>
       <script>
         setTimeout(() => {
           const app = document.getElementById("render-embed-here");
-          const container = document.createElement("div");
-          container.id = "fides-embed-container";
-          app.appendChild(container);
+          if (app) {
```

Comment:
one of our Cypress tests is overwriting the <body> which causes this to error out, so I'm adding a safeguard. Good idea anyway.